Services/BetQuoteService.cs

```diff
@@ -48,6 +49,45 @@ public async Task<BetQuoteDto> GetByIdAsync(Guid id)
             return entityDto;
         }
 
+        private BetQuotes GetNewQuotesByType(BetOptions type,  BetQuotes currentBetQuote)
+        {
+            if(type == BetOptions.A)
+            {
+                currentBetQuote.QuoteA += 0.01m;
+                currentBetQuote.QuoteX -= 0.01m;
+                currentBetQuote.QuoteB -= 0.01m;
+            }
+
+            if (type == BetOptions.B)
+            {
+                currentBetQuote.QuoteA -= 0.01m;
+                currentBetQuote.QuoteX -= 0.01m;
+                currentBetQuote.QuoteB += 0.01m;
+            }
+
+            if (type == BetOptions.X)
+            {
+                currentBetQuote.QuoteA -= 0.01m;
+                currentBetQuote.QuoteX -= 0.01m;
+                currentBetQuote.QuoteB += 0.01m;
+            }
+
+            return currentBetQuote;
+        }
+
+        public async Task<bool> UpdateBetQuotes(BetOptions type, Guid betQuoteId)
+        {
+            BetQuotes currentBetQuote = await _dbContext.BetQuotes.FindAsync(betQuoteId);
+            if(currentBetQuote == null)
+            {
+                return false;
+            }
+
+            currentBetQuote = GetNewQuotesByType(type, currentBetQuote);
```

Comment:
this doesn't make sense. the idea of having quotes saved in a different table was to be able to track the placed bets. If you change the quote value, how will you be able to track the placed bet
Also now I realize, we forgot the value in the placed bets table ...